scripts/new_project.kts

```diff
@@ -1,25 +1,72 @@
+#!/usr/bin/env kscript
+
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.1")
+@file:MavenRepository("central", "https://repo.maven.apache.org/maven2/")
+
+import kotlinx.coroutines.*
+import kotlin.coroutines.coroutineContext
+import java.io.File
+
 object NewProject {
 
+    private const val ARGUMENT_DELIMITER = "="
+    private const val COROUTINE_FOLDER_NAME = "CoroutineTemplate"
     private const val KEY_APP_NAME = "app-name"
     private const val KEY_PACKAGE_NAME = "package-name"
-    private const val ARGUMENT_DELIMITER = "="
+    private const val TEMPLATE_FOLDER_NAME = "coroutine"
```

Comment:
Hmmm, I think there was a misunderstanding here 🙇  My suggestion was to rename `COROUTINE_FOLDER_NAME` to `TEMPLATE_FOLDER_NAME` and remove line 16 as it's not being used anywhere 🙏